### ISSUE TITLE
@lucide/vue: `aria-hidden` is missing for icons

### ISSUE THREAD

**Issue by citruso** [lucide-icons/lucide#4564]
### Package

- [ ] lucide
- [ ] lucide-react
- [x] @lucide/vue
- [ ] @lucide/svelte
- [ ] @lucide/angular
- [ ] @lucide/astro
- [ ] lucide-react-native
- [ ] lucide-preact
- [ ] lucide-solid
- [ ] lucide-static
- [ ] @lucide/icons
- [ ] lucide-vue-next (deprecated)
- [ ] lucide-svelte (deprecated)
- [ ] lucide-angular (deprecated)
- [ ] Figma plugin
- [ ] source/main
- [ ] other/not relevant

### Version

1.25.0

### Can you reproduce this in the latest version?

- [x] Yes
- [ ] No

### Browser

- [ ] Chrome/Chromium
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] iOS Safari
- [ ] Opera
- [ ] Other/not relevant

### Operating system

- [ ] Windows
- [ ] Linux
- [ ] macOS
- [ ] ChromeOS
- [ ] iOS
- [ ] Android
- [ ] Other/not relevant

### Description

It seems that the `aria-hidden` property is missing from the source code.
Vue example (first code block): https://lucide.dev/guide/vue/basics/color
Any other example (first code block): https://lucide.dev/guide/lucide/basics/color

### Steps to reproduce

1. Import any icon
2. Check HTML output in devtools

### Checklist

- [x] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)

**Comment by karsa-mistmere** [comment]
I'm happy to report that #4409 will already resolve this.
